apps/web/pages/apps/installation/[[...step]].tsx

```diff
@@ -0,0 +1,509 @@
+    getDescription: (appName) => `Install ${appName} on your personal account or on a team account.`,
+    getStepNumber: () => 1,
+  },
+  [AppOnboardingSteps.EVENT_TYPES_STEP]: {
+    getTitle: () => "Select Event Type",
+    getDescription: (appName) => `On which event type do you want to install ${appName}?`,
+    getStepNumber: () => 2,
+  },
+  [AppOnboardingSteps.CONFIGURE_STEP]: {
+    getTitle: (appName) => `Configure ${appName}`,
+    getDescription: () => "Finalise the App setup. You can change these settings later.",
+    getStepNumber: () => 3,
+  },
+} as const;
+
+type OnboardingPageProps = {
+  hasTeams: boolean;
+  appMetadata: AppMeta;
+  step: StepType;
+  teams: TeamsProp;
+  personalAccount: PersonalAccountProps;
+  eventTypes?: TEventType[];
+  userName: string;
+  credentialId?: number;
+};
+
+const OnboardingPage = ({
+  hasTeams,
+  step,
+  teams,
+  personalAccount,
+  appMetadata,
+  eventTypes,
+  userName,
+  credentialId,
+}: OnboardingPageProps) => {
+  const pathname = usePathname();
+  const router = useRouter();
+
+  const [configureStep, setConfigureStep] = useState(false);
+  const [isSelectingAccount, setIsSelectingAccount] = useState(false);
+
+  const curerentStep: AppOnboardingSteps = useMemo(() => {
+    if (step == AppOnboardingSteps.EVENT_TYPES_STEP && configureStep) {
+      return AppOnboardingSteps.CONFIGURE_STEP;
+    }
+    return step;
+  }, [step, configureStep]);
+  const stepObj = STEPS_MAP[curerentStep];
+
+  const { t } = useLocale();
+  const utils = trpc.useContext();
+
+  const formPortalRef = useRef<HTMLDivElement>(null);
+
+  const formMethods = useForm<TEventTypesForm>({
+    defaultValues: {
+      eventTypes,
+    },
+  });
+
+  const updateMutation = trpc.viewer.eventTypes.update.useMutation({
+    onSuccess: async (data) => {
+      showToast(t("event_type_updated_successfully", { eventTypeTitle: data.eventType?.title }), "success");
+      router.push("/event-types");
+    },
+    async onSettled() {
+      await utils.viewer.eventTypes.get.invalidate();
+    },
+    onError: (err) => {
+      let message = "";
+      if (err instanceof HttpError) {
+        const message = `${err.statusCode}: ${err.message}`;
+        showToast(message, "error");
+      }
+
+      if (err.data?.code === "UNAUTHORIZED") {
+        message = `${err.data.code}: ${t("error_event_type_unauthorized_update")}`;
+        message = `${err.data.code}: ${t(err.message)}`;
+      }
+
+      if (err.data?.code === "INTERNAL_SERVER_ERROR") {
+        message = t("unexpected_error_try_again");
+      }
+
+      showToast(message ? t(message) : t(err.message), "error");
+      router.push("/apps");
+    },
+  });
+
+  const handleSelectAccount = async ({ id: teamId }: onSelectParams) => {
+    try {
+      setIsSelectingAccount(true);
+      if (appMetadata.isOAuth) {
+        const state = JSON.stringify({
+          appOnbaordingRedirectUrl: getAppOnboardingRedirectUrl(appMetadata.slug, teamId),
```

Comment:
```suggestion
          appOnboardingRedirectUrl: getAppOnboardingRedirectUrl(appMetadata.slug, teamId),
```
When fixing the typo here. Ensure that other parts of the app aren't expecting the typo'd value